msa-service/src/main/java/cz/muni/fi/pa165/msa/service/UserService.java

```diff
@@ -11,7 +11,9 @@
 @Service
 public interface UserService {
 
-    void createUser(User user);
+    void registerUser(User user, String password);
+
+    boolean authenticate(User u, String password);

```

Comment:
it would be nice to keep method argument names consistent.

Reply:
I agree with Miska, also one letter variable names does not look very good.